syntax = "proto3";

package message;

message Message {
  int64 room_id = 1;
  int64 message_id = 2;
  int64 author = 3;
  string text = 4;
}

service MessageService {
  rpc ChatStream (stream Message) returns (stream Message);
}